ksqldb-parser/src/test/java/io/confluent/ksql/parser/properties/with/CreateSourceAsPropertiesTest.java

```diff
@@ -149,10 +154,37 @@ public void shouldNotSetAvroSchemaNameForNonAvroKey() {
   public void shouldSetValidAvroSchemaName() {
     // When:
     final CreateSourceAsProperties properties = CreateSourceAsProperties.from(
-        ImmutableMap.of(CommonCreateConfigs.VALUE_AVRO_SCHEMA_FULL_NAME, new StringLiteral("schema")));
+        ImmutableMap.of(VALUE_AVRO_SCHEMA_FULL_NAME, new StringLiteral("schema")));
 
     // Then:
-    assertThat(properties.getValueFormatProperties().get(AvroFormat.FULL_SCHEMA_NAME), is("schema"));
+    assertThat(properties.getValueFormatProperties().get(ConnectProperties.FULL_SCHEMA_NAME), is("schema"));
+  }
+
+  @Test
+  public void shouldSetValueFullSchemaName() {
+    // When:
+    final CreateSourceAsProperties properties = CreateSourceAsProperties.from(
+        ImmutableMap.<String, Literal>builder()
+            .put(CommonCreateConfigs.VALUE_FORMAT_PROPERTY, new StringLiteral("Protobuf"))
+            .put(CommonCreateConfigs.VALUE_SCHEMA_FULL_NAME, new StringLiteral("schema"))
+            .build());
+
+    // Then:
+    assertThat(properties.getValueFormatProperties().get(ConnectProperties.FULL_SCHEMA_NAME), is("schema"));
+  }
+
+  @Test
+  public void shouldSetFullKeySchemaName() {
+    // Given:
+    final CreateSourceAsProperties props = CreateSourceAsProperties
+        .from(ImmutableMap.<String, Literal>builder()
+            .put(FORMAT_PROPERTY, new StringLiteral("Json_sr"))
+            .put(KEY_SCHEMA_FULL_NAME, new StringLiteral("KeySchema"))
+            .build());
+
+    // When / Then:
+    assertThat(props.getKeyFormatProperties("json_sr", "foo"),
+        hasEntry(ConnectProperties.FULL_SCHEMA_NAME, "KeySchema"));
```

Comment:
nit: why `hasEntry` compared to `...get(...), is(...)` as above?

Reply:
Good catch. Don't recall this. Will change previous one to `hasEntry`